website/content/docs/k8s/dns/views/enable.mdx

```diff
@@ -0,0 +1,115 @@
+---
+layout: docs
+page_title: Enable Consul DNS proxy for Kubernetes
+description: ->
+    Learn how to schedule a Consul DNS proxy for a Kubernetes Pod so that your services can return Consul DNS results for service discovery.
+---
+
+# Enable Consul DNS proxy for Kubernetes
+
+This page describes the process to deploy a Consul DNS proxy in a Kubernetes Pod so that Services can resolve Consul DNS requests. For more information, refer to [Consul DNS views for Kubernetes](/consul/docs/k8s/dns/views).
+
+## Prerequisites
+
+You must meet the following minimum application versions to enable the Consul DNS proxy for Kubernetes:
+
+- Consul v1.20.0 or higher
+- Either Consul on Kubernetes or the Consul Helm chart, v1.6.0 or higher
+
+## Update Helm values
+
+To enable the Consul DNS proxy, add the required [Helm values](/consul/docs/k8s/helm) to your Consul on Kubernetes deployment:
```

Comment:
```suggestion
To enable the Consul DNS proxy, add the required [Helm values](/consul/docs/k8s/helm) to your Consul on Kubernetes deployment.
```
Introduce code block with an descriptive, imperative sentence that ends with a period. [rule](https://github.com/hashicorp/tutorials/blob/main/STYLE_GUIDE.md#code-blocks)